src/OneExplorer/ConfigObject.ts

```diff
@@ -412,6 +447,15 @@ export class ConfigObj {
       })
     });
 
+    locatorRunner.register({
+      artifactAttr: {ext: '.json', icon: vscode.ThemeIcon.File, canHide: true},
+      locator: new Locator(
+          (value: string) => {
+            return LocatorRunner.searchWithCommandOption(value, '--save-chrome-trace', '.json');
```

Comment:
`--save-chrome-trace`
- I wonder if this option is `onecc` option or option of some backend tool. If this is option of backend tool, this option needs to be obtained from backend.

Reply:
Yes, I am working on that.
But, as you know, there are several backend PRs which are waiting for reviews and merge.
Therefore I decided to work on here and move these later.

Reply:
Could you write a `TODO` from the next time so that review does not review such code? Thanks.

Reply:
Okay :-D